dev-docs/adding-a-new-page-to-the-installer.md

```diff
@@ -0,0 +1,55 @@
+# Adding a new page to the installer
+
+## Add your page
+- Under `apps/ubuntu_bootstrap/lib/pages`, create a new directory for the page you would like to add
+ie: `apps/ubuntu_bootstrap/lib/pages/<your-feature>`
+- Create at minimum the following files:
+	- `<your-feature>_page.dart`: Defines the UI for the page you would like to add to the installer.
+	- `<your-feature>_model.dart`: Implements the provider and integrates with backend services to 
+    support your page's functionality.
+- Export your page for use in the installer by adding it to `apps/ubuntu_bootstrap/lib/pages.dart`
+- Add your page to the installation steps by adding it as an enum to `InstallationStep` in 
+`apps/ubuntu_bootstrap/lib/app/installation_step.dart`. The order of the `InstallationStep` enums 
```

Comment:
do we need to call out the weird behaviour of how page transitions can affect state here? @d-loose I still don't think I'm 100% sure I have my head round what the current behaviour is other than "here be dragons"

Reply:
Not exactly sure what you're referring to - do you mean the fact that pages come with a `load()` method that determines whether it's skipped or not (and additionally there's some logic in the `PageConfigService` that determines whether or not a page is shown based on the whitelabel.yaml)?

Reply:
Maybe I'm mis-remembering but I thought there was cakllbacks that fired to update state as you moved between pages and that in some places pages know (need to know?) what page is navigating to them.

Reply:
My understanding is the gotcha here is when you have sections with their own wizard that share a single model and state is persisted within that model between wizard steps? 

ie: `pages/storage/storage_wizard.dart`